# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/storage-map/.cargo-checksum.json
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/storage-map/.cargo-checksum.json b/third_party/rust/storage-map/.cargo-checksum.json
deleted file mode 100644
--- a/third_party/rust/storage-map/.cargo-checksum.json
+++ /dev/null
@@ -1,1 +0,0 @@
-{"files":{"Cargo.toml":"85bf5d8b5ab8ea2ba48dbd4523edb207c6462f342cd14ebb9a18ab94120a8c34","LICENSE":"c71d239df91726fc519c6eb72d318ec65820627232b2f796219e87dcf35d0ab4","README.md":"3c9074177c6dfe47137f5709f6d7590d2c10c07df505bb8b6a0e61d0cb6f12f6","rustfmt.toml":"e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855","src/lib.rs":"087d619e4ef4580ffd29ce8ca31b162ec93a9d148f3c32b880f3eba140853243","tests/ten.rs":"ee9028a5a5da0554014855cefb3a09ff6eedf4ca2f82dcc9332c9c13e5639b57"},"package":"418bb14643aa55a7841d5303f72cf512cfb323b8cc221d51580500a1ca75206c"}
\ No newline at end of file